src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/DebugLevelTests.cs

```diff
@@ -13,33 +13,121 @@
 
 namespace Wasm.Build.Tests.TestAppScenarios;
 
-public class DebugLevelTests : AppTestBase
+public class WasmSdkDebugLevelTests : DebugLevelTests
```

Comment:
I would prefer to have each class in a separate file, we can have them in a `DebugTests` dir. Other "scenario" files are 1 class only, max 100 lines, it's easier to read